lms/static/lms/js/require-config.js

```diff
@@ -19,6 +19,12 @@
         };
         defineDependency("jQuery", "jquery");
         defineDependency("_", "underscore");
+        if (window._ && window._.str) {
```

Comment:
Note to reviewers: underscore.string will already be loaded via base vendor file. This code will simplify with the underscore.string update because it no longer puts the library directly on "_".